javascript/sdk/lib/types.ts

```diff
@@ -399,3 +425,18 @@ export type PartnerEndpoints = Record<
     state: string;
   }
 >;
+
+export interface AuditLog {
+  name: string;
```

Comment:
just merged a PR (not yet deployed) that'll make this field named `event` instead of `name`

Reply:
thank you. updated!